import_export/resources.py

```diff
@@ -1082,9 +1095,17 @@ def get_fk_widget(cls, field):
         """
         Prepare widget for fk and o2o fields
         """
+
+        model = get_related_model(field)
+
+        use_natural_foreign_keys = ( 
+            has_natural_foreign_key(model) and cls._meta.use_natural_foreign_keys
```

Comment:
I wonder if we should raise an exception if the user has enabled ``use_natural_foreign_keys`` but not defined it for the model.  I guess this would always be an error?  It might be hard for them to track down otherwise.

Reply:
The thought wih this design is that if you have multiple models.ForeignKey fields and only some of them have natural key functions, your resource would still work (It would just not use the natural key functions when they don't exist). 

I think raising an exception would be reducing functionality because it'd require that all your foreign key fields have natural keys to work? The most likely scenario I can think of for wanting to mix and match is if you had a GUID primary key model (where the pk is the same in all environments) for some fields and natural keys for others. 

Not 100% sure though, just my take -- I like sensible defaults that reduce the configuration required. If we were doing a lot of logging it might make sense to WARN log there?

Reply:
> The thought wih this design is that if you have multiple models.ForeignKey fields and only some of them have natural key functions, your resource would still work (It would just not use the natural key functions when they don't exist).

Ah yes, that makes sense.